.github/workflows/build.yml

```diff
@@ -108,7 +108,7 @@ jobs:
         tests/airdrop_lamports.sh
 
         tests/test_solido.py
-        kill $validator
+        kill -9 $validator
```

Comment:
Aah, good idea! This might be why CI sometimes gets stuck!

Reply:
It's not because of this, I tried some other things with no success :|